hs_tracking/tests/test_tracking.py

```diff
@@ -95,7 +95,7 @@ def test_get(self):
         url_redirect = app_logging.get(r)
 
         # validate response
-        self.assertTrue(type(url_redirect) == response.HttpResponseRedirect)
+        self.assertTrue(type(url_redirect) is response.HttpResponseRedirect)
```

Comment:
What save action are you using to identify and change these?

Reply:
these showed up when I ran `flake8`:
```sh
➜ flake8
./hs_tracking/tests/test_tracking.py:98:25: E721 do not compare types, for exact checks use `is` / `is not`, for instance checks use `isinstance()`
```